src/utils/auth/isPasswordValid.ts

```diff
@@ -0,0 +1,10 @@
+export const isPasswordValid = (
+  password: string,
+  passwordConfirmation: string
+): boolean => {
+  return (
+    typeof password === "string" &&
```

Comment:
This is probably something outdated as I saw you [used](https://github.com/Code-the-Dream-School/ii-practicum-team-2-back/pull/7/files#diff-e4b4aaf425003cdc5308dee4dbf940634f05e381f77d339896074d115d7f479fR8) zod for these validations.

Reply:
removed